client/my-sites/stats/wordads/highlights-section.jsx

```diff
@@ -0,0 +1,154 @@
+import { useRef, useState } from 'react';
+import { useSelector } from 'react-redux';
+import { getWordAdsEarnings } from 'calypso/state/wordads/earnings/selectors';
+import HighlightCardSimple from './highlight-card-simple';
+
+// TODO: HighlightCard does not accept string values.
+// Should refactor to accept strings and move the business logic into the callers.
+// Then refactor this Comp to use HighlightCard again.
+
+function getAmountAsFormattedString( amount ) {
+	// Takes a Number, formats it to 2 decimal places, and prepends a "$".
+	// This mimics the existing behaviour. I'm assuming we only view/report
+	// on earnings in USD.
+	const formattedAmount = '$' + numberFormat( amount, 2 );
+	// Differs from previous behaviour in that we don't want "$0.00" as a result.
+	// Per design spec we'll return "$0" in this scenario.
+	// https://github.com/Automattic/wp-calypso/issues/72045
+	return formattedAmount === '$0.00' ? '$0' : formattedAmount;
+}
+
+function getHighlights( earnings ) {
+	const total = earnings && earnings.total_earnings ? Number( earnings.total_earnings ) : 0;
+	const owed = earnings && earnings.total_amount_owed ? Number( earnings.total_amount_owed ) : 0;
+	const paid = total - owed;
+
+	const highlights = [
+		{
+			heading: translate( 'Earnings', { comment: 'Total WordAds earnings to date' } ),
+			icon: <Icon icon={ payment } />,
+			amount: total,
+		},
+		{
+			heading: translate( 'Paid', {
+				comment: 'Total WordAds earnings that have been paid out',
+			} ),
+			icon: <Icon icon={ receipt } />,
+			amount: paid,
+		},
+		{
+			heading: translate( 'Outstanding amount', {
+				comment: 'Total WordAds earnings currently unpaid',
+			} ),
+			icon: <Icon icon={ tip } />,
+			amount: owed,
+		},
+	];
+	// Transform/index the data for use with React.
+	return highlights.map( ( highlight, i ) => ( {
+		id: i,
+		formattedAmount: getAmountAsFormattedString( highlight.amount ),
+		...highlight,
+	} ) );
+}
+
+function payoutNotices( earnings ) {
+	const owed =
+		earnings && earnings.total_amount_owed ? numberFormat( earnings.total_amount_owed, 2 ) : '0.00';
+	const notice = {
+		id: 'notice',
+		value: translate(
+			'Outstanding amount of $%(amountOwed)s does not exceed the minimum $100 needed to make the payment.',
+			{
+				comment: 'WordAds: Insufficient balance for payout.',
+				args: { amountOwed: owed },
+			}
+		),
+	};
+	const limit = {
+		id: 'limit',
+		value: translate(
+			'Payment will be made as soon as the total outstanding amount has reached $100.',
+			{
+				comment: 'WordAds: Payout limit description.',
+			}
+		),
+	};
+	const payout = {
+		id: 'payout',
+		value: translate(
+			'Outstanding amount of $%(amountOwed)s will be paid approximately 45 days following the end of the month in which it was earned.',
+			{
+				comment: 'WordAds: Payout will proceed.',
+				args: { amountOwed: owed },
+			}
+		),
+	};
+	return owed < 100 ? [ notice, limit ] : [ payout ];
```

Comment:
`owed` is a string and `100` is a number. It's generally not safe to compare strings to numbers - in this case it runs okay but I'd still recommend not doing this.

Reply:
I'll fix this up to. I didn't realize I was doing this.